### ISSUE TITLE
Unable to run minishift openshift config set --patch for Openshift 3.9

### ISSUE THREAD

**Issue by geoand** [minishift/minishift#2156]
### General information

  * Minishift version: minishift v1.14.0+1ec5877
  * OS: Linux
  * Hypervisor: KVM
  * Openshift version: v3.9.0-alpha.3

### Steps to reproduce

`minishift openshift config set --target node --patch '{"kubeletArguments": { "max-pods": ["30"] } }'`

### Expected

Minishift should patch the node configuration file and restart

### Actual

Minishift is unable to perform the patch

The output is:

```
E0320 16:50:48.671795   13486 openshift.go:96] Creating patched configuration failed. Not applying the changes.ssh command error:
command : docker exec -t origin /usr/bin/openshift ex config patch /var/lib/origin/openshift.local.config/node-localhost/node-config.yaml --patch='{"kubeletArguments": { "max-pods": ["30"] } }'
err     : exit status 1
output  : Error: unknown command "ex" for "openshift"
Run 'openshift --help' for usage.
```

### More info

I need to add that the patch operation works just fine when running it on a minishift VM that has Openshift v3.7.1 deployed.

**Comment by geoand** [comment]
Thank you for the information!

Will there be some kind of replacement or workaround implemented in
Minishift?

**Comment by anjannath** [comment]
Yes, we will be looking into the new version and see what things have changed and accordingly update our code base to cater to the new changes, also keeping in mind backward compatibility. But, for now as a work around you can patch the `node-config.yaml` manually using `oc ex config patch`, for that you have to ssh into the vm.

```
# first ssh into the vm
$ minishift ssh

# patch the config file
$ docker exec -t origin /usr/bin/oc ex config patch /var/lib/origin/openshift.local.config/node-localhost/node-config.yaml --patch '{"ku
beletArguments": { "max-pods": ["30"] } }' > config
$ cat config | sed "s/'/\'/" | sed "s/.$//" | sudo tee /var/lib/minishift/openshift.local.config/node-localhost/node-config.yaml > /dev/null

# restart origin container
$ docker restart origin
```

**Comment by geoand** [comment]
Good to know, thanks again!

**Comment by LalatenduMohanty** [comment]
Resolved by #2161
